addons/mail/static/src/utils/test_utils.js

```diff
@@ -807,6 +807,7 @@ async function start(param0 = {}) {
     await waitUntilEventPromise;
     return {
         ...result,
+        afterNextRender,
```

Comment:
Bit strange to have 2 ways to retrieve it (`import { afterNextRender } from '@mail/utils/test_utils'` and `const { afterNextRender } = start()`). But TBH I'm quite indifferent with either choices.

Reply:
The goal would be to get rid of the import and only use the one from start, as those helpers often need to depend on the test setup itself. For now it is not the case for `afterNextRender`, but it might become the case one day so better prepare it now and be consistent with the others.